index.js

```diff
@@ -19,8 +19,8 @@ const HOSTNAME_IP_REGEX = /[^0-9a-zA-Z\-.]/g;
 const HOSTNAME_PATH = '/etc/logdna-hostname';
 
 // Variables
-var config = require('./lib/config');
-var processed;
+let config = require('./lib/config');
```

Comment:
Would this be better as a `const`?

Reply:
No, @jakedipity, it gets reassigned in the bottom - I think I tried to make it `const` before but failed every time since it requires some rearrangements of the lines

Reply:
oof, it's *really* weird to mutate something that came from `require`.  Can you re-work it?